### DISCUSSION TITLE
I want to update field value by clicking on button (plus,minus). what to do?

### DISCUSSION THREAD

**Discussion by deepakchandrawanshi1063** [react-hook-form/react-hook-form#8780] [Q&A] [answered]
![buttons](https://user-images.githubusercontent.com/41387081/182386383-f85294e0-868f-4178-a506-4d30e64e4c26.png)

https://codesandbox.io/s/upbeat-curran-d2jjzm?file=/src/App.js

**Comment by bluebill1049** [comment, marked as answer]
please provide a codesandbox (I wrote this 100times at least), consider if someone else send you code as above.

**Reply by deepakchandrawanshi1063** [reply]
> please provide a codesandbox (I wrote this 100times at least), consider if someone else send you code as above.

ok

**Reply by deepakchandrawanshi1063** [reply]
sir help me. I created a sandbox with buttons.

**Reply by bluebill1049** [reply]
could you just `useController` for this? you can use the onChange to update the field value 

```tsx
const { onChange, value } = useController({ name: 'test' })

<input onChange={onChange} />

<button onClick={() => {
  onChange(value + 1)
}} />
```

**Reply by deepakchandrawanshi1063** [reply]
Thank you sir. it's working.
